travel-event-host-app/src/components/categories/categories.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from './styles.module.css';
+import categoriesArr from '@/lib/categoryArray';
+
+export default function Categories() {
+  return (
+    <section className={styles.section}>
+      <h2>Categories</h2>
+      <ul className={styles.categoriesUl}>
+        {categoriesArr.map((category) => (
```

Comment:
re: my comment below about making the categories array a dictionary, you can get the values as an array of strings using `Object.values(CategoryDict)` and then `map` them from there.

Reply:
why use that, Typescript things?

Reply:
ok